src/compass/s1_geocode_slc.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-'''wrapper for geocoded SLC'''
+'''wrapper for geocoded CSLC'''
```

Comment:
```suggestion
'''wrapper for geocoded SLC'''
```

Geocoded SLC is more commonly used and accepted term, I would suggest keeping the old name.

Reply:
In discussions in #75, @vbrancat suggested [`CSLC`](https://github.com/opera-adt/COMPASS/blob/a5138d09e37c60223b1ed6259d652d8bdacf15ad/src/compass/s1_geocode_slc.py#L137).

Reply:
I was not aware of it. Thanks @LiangJYu